org.eclipse.dawnsci.nexus/src/org/eclipse/dawnsci/nexus/NexusFile.java

```diff
@@ -29,9 +29,10 @@
 public interface NexusFile extends AutoCloseable {
 
 	/**
-	 * @deprecated Use {@link NexusConstants#CLASS}
+	 * @deprecated Use {@link NexusConstants#NXCLASS}
 	 */
-	public static final String NXCLASS = NexusConstants.CLASS;
+	@Deprecated
+	public static final String NXCLASS = NexusConstants.NXCLASS;
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)